### DISCUSSION TITLE
MacBook Pro 14 M1 Pro , MacOS Ventura USB-C problem and iiyama XUB3293UHSN monitor

### DISCUSSION THREAD

**Discussion by backrzysztof** [MonitorControl/MonitorControl#1277] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

After uploading MonitorControl and upgrading MacOS to version 13, the monitor after turning off the computer freezes when I had the Montery 12 version, the problem occurred sporadically, now after turning off and on the computer, the monitor does not start up and I have to restart it with mains power. After uninstalling Monitor Control everything works fine.

### Expected behavior

A proposal to solve the problem

### Environment Information (please complete the following information)

```markdown
- macOS version: 13 Ventura
- Mac model: Apple M1 Pro (14-inch,2021)
- MonitorControl version: v4.1.0
- Monitor(s): IIYAMA XUB3293UHSN
- Monitor Cable(s)/Connection(s): USBC DP + PD
- Apple Silicon/M1 (yes or no): YES
```

**Comment by waydabber** [comment]
Hi there - you might want to try to disable DDC and see if that solves the problem. Let me know if this helps!

**Comment by backrzysztof** [comment]
The problem persists, I noticed that the monitor wakes up when the computer sleeps and then goes to sleep after a while
